packages/l/libcurl/versions.lua

```diff
@@ -0,0 +1,45 @@
+add_versions("7.73.0", "cf34fe0b07b800f1c01a499a6e8b2af548f6d0e044dca4a29d88a4bee146d131")
```

Comment:
Please modify as I said, you can only use this way to includes it.

```
includes("versions.lua")
package("curl")
    add_urls()
    add_versions_list()
versions.lua

function add_versions_list()
end
```

Reply:
I did this to prevent polluting the global namespace (I was writing my comment) and letting a `add_versions_list` function exist / being overridden or worse.